src/post_setup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
We typically set up our bash scripts in this fashion. Is there any reason this would be problematic for this script?

```suggestion
#!/bin/env bash

set -o nounset
set -o errexit
set -o pipefail
```

Reply:
I agree, POSIX compliance is the way.